packages/types/src/lib/interfaces.ts

```diff
@@ -300,6 +309,31 @@ export interface JsonSignChainDataRequest {
   exp: number;
 }
 
+export interface JsonSignSessionKeyRequestV1 {
+  sessionKey: string;
+  authMethods: AuthMethod[];
+  pkpPublicKey?: string;
+  authSig?: AuthSig;
+  siweMessage: string;
+  curveType: 'BLS' | 'ECDSA';
+  code?: string;
+  litActionIpfsId?: string;
+  jsParams?: any;
+  epoch?: number;
+}
+
+export interface BlsResponseData {
+  result: boolean;
+  signatureShare: {
+    ProofOfPossession: string;
+  };
+  shareIndex: number;
+  curveType: string;
+  siweMessage: string;
+  dataSigned: string;
+  blsRootPubkey: string;
+}
```

Comment:
nice, we have a new model for this in the wasm combination branch but for now this is better than what we have. making a note to unify with the wasm combination work.